gliderpy/fetchers.py

```diff
@@ -90,25 +90,20 @@ def plot_track(self):
         ax.coastlines("10m")
         ax.set_extent([x.min() - dx, x.max() + dx, y.min() - dy, y.max() + dy])
         return fig, ax
-    
-    def plot_transect(self,var):
+
+    def plot_transect(self, var):
+        import matplotlib.dates as mdates
+
         """
         Makes a scatter plot of depth vs time coloured by a user defined variable
         :param var: variable to colour the scatter plot
         :return: figure, axes
         """
-        
+
         idx = self.to_pandas()
-        cmap = Haline_20.mpl_colormap
```

Comment:
@LindsayRAbrams I removed the salinity colormap for now and opened #21 so we can handle this more gracefully later. Let's leave the default cmap for the presentation.